### ISSUE TITLE
Support official Pi debug probe 

### ISSUE THREAD

**Issue by thejpster** [korken89/pico-probe#5]
https://www.raspberrypi.com/news/raspberry-pi-debug-probe-a-plug-and-play-debug-kit-for-12/

Should just involve setting some pin assignments?

**Comment by korken89** [comment]
Indeed, checking the schematics it seems to set the SWDIO/SWDCLK to GPIO 14/12 and disable the support for RST should be enough. :)
